### ISSUE TITLE
[TF2] Taunt Wheel

### ISSUE THREAD

**Issue by gaelcoral** [ValveSoftware/Source-1-Games#6809]
With every TF2 update, new taunts are added, making it impossible to equip all the taunts some classes offer. It’s time to replace the current taunt selection system with a taunt wheel.
This change would allow players to have taunt pages instead of being limited to only 8. Additionally, for players using a controller or Steam Deck, it would be much more convenient to choose a taunt with the wheel rather than selecting them one by one.
I feel this change would be beneficial for all players and would significantly improve the game’s QoL.
![image](https://github.com/user-attachments/assets/a7087b91-306b-4854-8591-1407b8c24dff)
_(A quick mockup I made)_

**Comment by KimmyTF2** [comment]
Yes, yes, yes, and one more yes. With only 8 slots and so many taunts in the game nowadays, going back and forth changing them in the taunt menu is tedious.

**Comment by Chickenman456** [comment]
I saw your comment and thought I might as well post my proposal as well.

Workshop link: https://steamcommunity.com/sharedfiles/filedetails/?id=3441735176&tscn=1741629265

![Image](https://github.com/user-attachments/assets/59dd4207-ea2a-4f9c-bdc6-57dec66591c7)
![Image](https://github.com/user-attachments/assets/8527b566-b580-42b1-87a5-25e51257282c)
![Image](https://github.com/user-attachments/assets/87def69d-f8b6-4f78-8167-b18ba2a9a5c8)
![Image](https://github.com/user-attachments/assets/a8d60f95-b4f6-4a3f-8ed8-5240a45add0a)
![Image](https://github.com/user-attachments/assets/fbb14dd1-b4f4-42b9-889b-371a34a24af9)
![Image](https://github.com/user-attachments/assets/f4fbbe60-9481-4231-9654-47036204a65c)
